assets/FcmPushAsset.php

```diff
@@ -2,11 +2,12 @@
 
 namespace humhub\modules\fcmPush\assets;
 
-use humhub\modules\fcmPush\Module;
-use humhub\modules\fcmPush\services\DriverService;
 use Yii;
 use yii\helpers\Url;
 use yii\web\AssetBundle;
+use humhub\widgets\Button;
```

Comment:
Is this really required?

Reply:
I forgot to remove it due to using the view instead of the assets class.